Postman/Nfield Public API.postman_collection.json

```diff
@@ -11724,7 +11728,7 @@
 									"response": []
 								},
 								{
-									"name": "Get Sample Data (deprecated)",
+									"name": "Get Sample Data",
```

Comment:
It seems like Get Sample Data (deprecated) or Get Sample Data is removed.
I think they both should still be there.

Reply:
Maybe there were things merged to master that you didn't have. it would be better if you remove your postman collection, import the latest one from master, add again the new endpoint and export it